modules/programs/fish.nix

```diff
@@ -214,14 +214,6 @@ in {
           initialisation.
         '';
       };
-
-      promptInit = mkOption {
```

Comment:
You will need to add a `mkRemovedOptionModule` import to deprecate the option.

Reply:
Borrowed description from https://github.com/nix-community/home-manager/blob/5f6364fc28065d6b26423512e14207ce0bffb27a/modules/services/udiskie.nix#L24-L28, is it OK?